### ISSUE TITLE
Process finished with exit code -1073741676 (0xC0000094)

### ISSUE THREAD

**Issue by sz547073165** [PaddlePaddle/ERNIE#362]
在pycharm，用自己数据预训练，执行[train.py](https://github.com/PaddlePaddle/ERNIE/blob/develop/ernie/train.py)到一半的时候终止了，出现了这个code。

有时候能执行完，有时候强制中断，执行的程度不好说，有人碰到过类似的吗？

paddlepaddle 1.5.2
Python 3.7.4

如：

`
[INFO] 2019-11-05 18:59:32,298 [    train.py:  318]:	feed_queue size 70
[INFO] 2019-11-05 18:59:32,298 [    train.py:  323]:	current learning_rate:0.000095
[INFO] 2019-11-05 18:59:32,298 [    train.py:  343]:	epoch: 1, progress: 1/1, step: 510, loss: 5.830832, ppl: 287.308990, next_sent_acc: 1.000000, speed: 0.431771 steps/s, file: C:/Users/Admin/Desktop/work/20190920nlp/nlp01/dataset2/pre_data_train.tsv.gz, mask_type: mask_word
[INFO] 2019-11-05 19:00:05,023 [    train.py:  318]:	feed_queue size 70
[INFO] 2019-11-05 19:00:05,024 [    train.py:  323]:	current learning_rate:0.000095
[INFO] 2019-11-05 19:00:05,024 [    train.py:  343]:	epoch: 1, progress: 1/1, step: 520, loss: 5.694308, ppl: 212.865204, next_sent_acc: 1.000000, speed: 0.305581 steps/s, file: C:/Users/Admin/Desktop/work/20190920nlp/nlp01/dataset2/pre_data_train.tsv.gz, mask_type: mask_word

Process finished with exit code -1073741676 (0xC0000094)
`

**Comment by GT-ZhangAcer** [comment]
这个应该是CPU模式训练吧，内存直接爆掉了。可以调低Batch_size或者max_len试试。

**Comment by sz547073165** [comment]
> 这个应该是CPU模式训练吧，内存直接爆掉了。可以调低Batch_size或者max_len试试。

感谢 我试试看
